.github/workflows/publish-docs-on-release.yml

```diff
@@ -1,14 +1,15 @@
-name: Build and Deploy Docs
+name: Deploy Documentation on Release
 
 on:
   release:
-    types:
-      - published
+    types: [published]
```

Comment:
The correct syntax

Ref: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release